### PR TITLE
Update web components to USWDS v3

### PR DESCRIPTION
**Pull request by jenniemc**
## Summary

This PR updates the following web components from USWDS v1 to v3.
- [ ] Additional Info `<va-additional-info>`
- [ ] Alert `va-alert`
- [ ] Modal `VaModal`
- [ ] Button `<va-button>`

The Breadcrumb `<VaBreadcrumbs>` will be updated on a separate ticket since additional effort is needed to have the component behave as expected.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#74340

## Testing done

unit and e2e tests

## Screenshots

| Before | After |
| ------ | ----- |
|     
Screenshot 2024-01-30 at 7 07 12 PM |    
Screenshot 2024-01-29 at 6 43 36 PM |
| Screenshot 2024-01-30 at 7 06 01 PM  |   
Screenshot 2024-01-29 at 7 32 30 PM |
Screenshot 2024-01-30 at 7 05 16 PM |
Screenshot 2024-01-30 at 3 14 22 PM

## What areas of the site does it impact?

Web components

## Acceptance criteria
The component below are converted to V3
- [ ] Additional Info `<va-additional-info>`
- [ ] Alert `va-alert`
- [ ] Modal `VaModal`
- [ ] Button `<va-button>`

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/cancel/CancelAppointmentSucceededModal.jsx [hunk 1]
```diff
@@ -17,15 +17,13 @@ export default function CancelAppointmentSucceededModal({
         modalTitle={`Your ${typeText} has been canceled`}
         data-testid={`cancel-${typeText}-SuccessModal`}
         role="alertdialog"
+        primaryButtonText="Continue"
+        onPrimaryButtonClick={onClose}
+        uswds
```
Comment (jenniemc) [inline src/applications/vaos/appointment-list/components/cancel/CancelAppointmentSucceededModal.jsx:22]: Replacing the button element to utilize `<VaModal>`'s `primaryButtonText` property and `onPrimaryButtonClick` custom event

File: src/applications/vaos/components/VAOSApp/DowntimeMessage.jsx [hunk 2]
```diff
@@ -70,6 +70,9 @@ export default function DowntimeMessage({
             'VA online scheduling will be down for maintenance'
           }
           data-testid="downtime-approaching-modal"
+          secondaryButtonText="Dismiss"
+          onSecondaryButtonClick={close}
+          uswds
```
Comment (jenniemc) [inline src/applications/vaos/components/VAOSApp/DowntimeMessage.jsx:75]: Replacing the button element to utilize `<VaModal>`'s `secondaryButtonText` property and `onSecondaryButtonClick` custom event

File: src/applications/vaos/new-appointment/components/TypeOfCarePage/PodiatryAppointmentUnavailableModal.jsx [hunk 3]
```diff
@@ -12,12 +12,16 @@ const PodiatryAppointmentUnavailableModal = ({ onClose, showModal }) => {
...
+      data-testid="toc-modal"
       status="warning"
       visible
       onCloseEvent={onClose}
       modalTitle={title}
       ariaLabel={title}
       role="alertdialog"
+      primaryButtonText="Ok"
+      onPrimaryButtonClick={onClose}
+      uswds
```
Comment (jenniemc) [inline src/applications/vaos/new-appointment/components/TypeOfCarePage/PodiatryAppointmentUnavailableModal.jsx:24]: Replacing the button element to utilize `<VaModal>`'s `primaryButtonText` property and `onPrimaryButtonClick` custom event

**Review by vbahinwillit — COMMENTED** [review]
LGTM!

**Comment by simiadebowale** [comment]
@jenniemc - LGTM, I want Peter & the UX team to review the style changes before we merge

**Comment by melissa-lefevre** [comment]
The UI in the before and after looks good.

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**